spec/features/user_can_checkout_spec.rb

```diff
@@ -11,7 +11,18 @@
       add_item_to_cart
       visit cart_path(user_one)
       fill_in "days", with: 5
-      click_button "Checkout"
+      click_button "Pay with Card"
+
+      sleep(2)
+      stripe_iframe = all('iframe[name=stripe_checkout_app]').last
+      Capybara.within_frame stripe_iframe do
+        page.execute_script(%Q{ $('input#email').val('jamesdd9302@yahoo.com'); })
+          page.execute_script(%Q{ $('input#card_number').val('4242424242424242'); })
+          page.execute_script(%Q{ $('input#cc-exp').val('08/44'); })
+          page.execute_script(%Q{ $('input#cc-csc').val('999'); })
+          page.execute_script(%Q{ $('#submitButton').click(); })
```

Comment:
Use `%` instead of `%Q`.<br>Inconsistent indentation detected.<br>Do not use `%Q` unless interpolation is needed.  Use `%q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.